src/cmake/axom-config.cmake.in

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+@PACKAGE_INIT@
+
+if(NOT AXOM_FOUND)
+
+  #----------------------------------------------------------------------------
+  # Set version and paths
+  #----------------------------------------------------------------------------
+  
+  set(AXOM_VERSION "@AXOM_VERSION_FULL@")
+  
+  set(AXOM_INSTALL_PREFIX "@AXOM_INSTALL_PREFIX@")
+  set(AXOM_INCLUDE_DIRS "${AXOM_INSTALL_PREFIX}/include/axom")
+
+  #----------------------------------------------------------------------------
+  # User configuration options and features
+  #----------------------------------------------------------------------------
+
+  set(AXOM_ENABLE_FORTRAN     "@ENABLE_FORTRAN@")
+
+  set(AXOM_ENABLE_LUMBERJACK  "@AXOM_ENABLE_LUMBERJACK@")
+  set(AXOM_ENABLE_SLIC        "@AXOM_ENABLE_SLIC@")
+  set(AXOM_ENABLE_SLAM        "@AXOM_ENABLE_SLAM@")
+  set(AXOM_ENABLE_MINT        "@AXOM_ENABLE_MINT@")
+  set(AXOM_ENABLE_PRIMAL      "@AXOM_ENABLE_PRIMAL@")
+  set(AXOM_ENABLE_SPIN        "@AXOM_ENABLE_SPIN@")
+  set(AXOM_ENABLE_SIDRE       "@AXOM_ENABLE_SIDRE@")
+  set(AXOM_ENABLE_QUEST       "@AXOM_ENABLE_QUEST@")
```

Comment:
I suggest we keep this list sorted alphabetically.

Reply:
But i like being first.... makes me feel important

Reply:
If you are referring to `LUMBERJACK` appearing first on the list, that would still be the case after the list is sorted! So, there you go!

Reply:
! i swear i know my alphabet! Though my next component will be AAA (Absolutely Awesome Acronym)